Alexey/Change Return Program.py

```diff
@@ -0,0 +1,28 @@
+
+def main():
+    total = float(input("How much money you need to change: ")) * 100

```

Comment:
I doesn't work 
Test:
How much money you need to change: 7
Quarters: 28 
Dimes: 0 
Nickels: 0
Pennies: 0

Reply:
Sorry, I have not specified input in dollars, 7 dollars its 700 cents and 28 of Quarters